src/ui-kit/components/Input/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input as SemanticInput } from 'semantic-ui-react';
+import styled from 'styled-components';
+import { brandPrimaryColor, warningColor } from '../../theme';
+
+const PrimaryInput = styled(SemanticInput)`
+  &&& {
+    &&& input {
+      &:focus {
+        box-shadow: 0 0 5px ${brandPrimaryColor};
+      }
+    }
+  }
+`;
+
+const InputWithError = styled(SemanticInput)`
+  &&& {
+    &&& input {
+      color: ${warningColor};
+      border-style: solid;
+      border-width: 1px;
+      border-color: ${warningColor};
+
+      &:focus {
+        box-shadow: 0 0 5px ${warningColor};
+      }
+    }
+  }
+`;
+
+const Input = props => {
+  const { error, ...rest } = props;
+  if (error) return <InputWithError {...rest} />;
+
+  return <PrimaryInput {...props} />;
+};
+
+Input.propTypes = {
+  error: PropTypes.bool,
+};
+
+export default Input;
```

Comment:
Could we also name ui kit components not index.js but `Input.js` and so on?

It's confusing to see millions of index.js files everywhere tbh

Reply:
So, In this case, we should import ui-kit components in the following style:
import Input from '../ui-kit/components/Input/Input';
And it will be confusing too